R/fftrees_ffttowords.R

```diff
@@ -110,18 +118,27 @@ fftrees_ffttowords <- function(x = NULL,
         direction.pos.i <- directions.v[i]
 
         if (classes.v[i] %in% c("c", "l")) {
+
+          # negate direction:
           sentence.i.1 <- paste0(
             "If ", cues.v[i], " ", direction.neg.i, " {", thresholds.v[i], "}, decide ",
             decision.labels[1]
           )
 
+          # # NOT negate direction:
```

Comment:
I think the bug of the final node (when creating an FFT from verbal description) is due to lines 112 to 165 here.  

I removed the commented line, but currently don't yet understand why the cue direction is being negated there (marked as `WHY???` in the code).  Somehow, the reversal of directions must be conditionalised on exit types (0 vs 1)?

(So the code in current PR works, but does not yet fix the error.)

Reply:
**Update**:  The latest commits should fix the **final node bug** by making 2 changes:

1. `fftrees_wordstofftrees()` now ignores anything after `"otherwise"` (i.e., the 2nd part of the last sentence of a verbal FFT description).  (This prevents the final cue direction from _always_ being reversed.)   

2. In the final sentence ("If A, otherwise B"), `fftrees_ffttowords()` now always predicts/decides positive criterion instances (TRUE cases in A) _prior to_ negative criterion instances (FALSE cases in B).  This renders additional reversals of cue directions obsolete. 

Some initial checks are included in the `FFTrees_mytree.Rmd` vignette, but further checks should be done.